data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            label: "sandbag",
+        },
+        description: {
+            content: "This registry visualizes REDD+ initiatives in members of the [Coalition of Rainforest Nations](https://www.rainforestcoalition.org)."
+            + " It shows which countries (without rainforests) are funding these initiatives. The data doesn't come with dates and seems a bit abandoned overall."
+            + " It's hosted by the FAO (Food and Agriculture Organization at the UN)."
+            + " Also there are [serious concerns about REDD+](https://en.wikipedia.org/wiki/Reducing_emissions_from_deforestation_and_forest_degradation#Concerns).",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "C2ES",
+            link: "https://www.c2es.org/",
+            keywords: ["legislation", "think-tank", "policy", "targets-private-sector"],
+            label: "c2es",
+        },
+        description: {
+            content: "The Center for Climate and Energy Solutions prepares and pushes for strong climate policy reduce greenhouse gas emissions, promote clean energy, and strengthen resilience to climate impacts."
+            + " It works internationally but has a big focus on US federal, state and municipial levels."
+            + " The Center also consults businesses to become more sustainable.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[1998](https://www.c2es.org/about/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "The Climate Registry",
+            link: "https://www.theclimateregistry.org",
+            keywords: ["reporting", "measuring", "modelling", "targets-private-sector", "ngo"],
+            label: "the-climate-registry",
+        },
+        description: {
+            content: "The Climate Registry (TCR) runs greenhouse gas reporting programs globally that allow organizations to measure, report and verify the carbon in their operations in order to manage and reduce it."
+            + " TCR also consults with governments on all aspects of greenhouse gas measurement, reporting, and verification."
+            + " The Center also consults businesses to become more sustainable.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+        established: {
+            content: "[2007](https://www.theclimateregistry.org/who-we-are/about-us/), superseeding the [California Climate Action Registry](http://www.caclimateregistry.org/) which was established in 2001",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "EIT Climate-KIC",
+            link: "https://www.climate-kic.org",
+            keywords: ["think-tank", "network", "financing", "ppp"],
```

Comment:
add education

Reply:
Let's discuss tags in the sheet, the tags here are outdated

Reply:
ok we need a pool of tags we are going to use. Can you provide one with all you used already so we can collect new ones and discus on them. Please share document